AppConnectSwift/CreateAccountViewController.swift

```diff
@@ -18,16 +18,24 @@ class CreateAccountViewController: UIViewController {
     @IBAction func createAccount(sender: AnyObject) {
         let clientFactory = MDClientFactory.sharedInstance()
         let client = clientFactory.clientOfType(MDClientType.Network);
-        client.registerSubjectWithEmail(userEmail, password: userPassword, securityQuestionID: userSecurityQuestionID, securityQuestionAnswer: securityQuestionLabel.text) { (err) in
-            if err == nil {
-                NSOperationQueue.mainQueue().addOperationWithBlock({ 
-                    self.performSegueWithIdentifier("CreateAccountSuccess", sender: nil)
-                })
-            }
-            else {
-                print(err?.description)
+        if userSecurityQuestionAnswer.text?.characters.count >= 2 {
+            client.registerSubjectWithEmail(userEmail, password: userPassword, securityQuestionID: userSecurityQuestionID, securityQuestionAnswer: userSecurityQuestionAnswer.text) { (err) in
+                if err == nil {
+                    NSOperationQueue.mainQueue().addOperationWithBlock({
+                        self.showAlert("Account Creation Success", message: "Created account successfully")
+                        self.performSegueWithIdentifier("CreateAccountSuccess", sender: nil)
+                    })
+                }
+                else {
+                    print(err?.description)
+                    self.showAlert("Account Creation Failure", message: "\(err!)" )
```

Comment:
Actually `description` will work just fine too - and since it's now shown in the alert, the `print` statement can be removed.